src/library/Payment/Adapter/PayPalEmail.php

```
@@ -58,7 +59,7 @@ public static function getConfig(): array
 
     public function getHtml($api_admin, $invoice_id, $subscription): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$api_admin'. (undefined)

(UnusedFormalParameter)